playwright-e2e/dss/component/checkbox.ts

```diff
@@ -34,20 +34,29 @@ export default class Checkbox extends WidgetBase {
     return isChecked ? isChecked : false;
   }
 
-  async check(opts: { timeout?: number } = {}): Promise<void> {
-    const { timeout } = opts;
+  async check(opts: { timeout?: number, callback?: () => any } = {}): Promise<void> {
```

Comment:
Sometimes, click triggers a `/patch` request. waiting for that request to finish is important in order to prevent any flaky test. New `callback` parameter can be used to wait for any triggered request.